services/motion/builtin/move_request.go

```diff
@@ -92,6 +93,36 @@ func (mr *moveRequest) Plan(ctx context.Context) (state.PlanResponse, error) {
 	}
 	mr.planRequest.StartConfiguration = map[string][]referenceframe.Input{mr.kinematicBase.Kinematics().Name(): inputs}
 
+	existingGifs, err := mr.planRequest.WorldState.ObstaclesInWorldFrame(
+		mr.planRequest.FrameSystem,
+		referenceframe.StartPositions(mr.planRequest.FrameSystem),
```

Comment:
hmmmmm, thinking about this a little, I want to ask: wouldn't the output be the same for both at this point?

-----
doing `referenceframe.StartPositions(mr.planRequest.FrameSystem)` we know we will have a zero'd output always.

If we were to use `fsService.CurrentInputs` then i think we would have have a zero'd output since we haven't moved. Conversely, even if we have moved our current inputs for a base should go back to zero since we cancelled context.

Reply:
>If we were to use fsService.CurrentInputs then i think we would have have a zero'd output since we haven't moved.

That is only the case if we are asking where `base` is relative to `world`.

If we are asking about a camera mounted on a arm/gantry on a base, then `CurrentInputs` will give us that. Zeroed inputs will give us bad data.

We can't move a base/arm simultaneously right now, but you could absolutely put an arm on a base, move them one at a time, and know where a camera mounted on the arm is based on current joint positions.

Reply:
Or more specifically, this should use `mr.planRequest.StartConfiguration`